cmd/thriftbreak/main_test.go

```diff
@@ -60,30 +88,67 @@ func TestThriftBreakIntegration(t *testing.T) {
 		"somefile.go": `service Qux{}`,
 	}
 	remove := []string{"test/d.thrift"}
-	breaktest.CreateRepoAndCommit(t, tmpDir, from, to, remove)
 
-	f, err := ioutil.TempFile(tmpDir, "stdout")
-	require.NoError(t, err, "create temporary file")
-	defer func(oldStdout *os.File) {
-		assert.NoError(t, f.Close())
-		os.Stdout = oldStdout
-	}(os.Stdout)
-	os.Stdout = f
+	for _, tt := range tests {
+		t.Run(tt.desc, func(t *testing.T) {
+			tmpDir := t.TempDir()
+			breaktest.CreateRepoAndCommit(t, tmpDir, from, to, remove)
+
+			f, err := ioutil.TempFile(tmpDir, "stdout")
+			require.NoError(t, err, "create temporary file")
+			defer func(oldStdout *os.File) {
+				assert.NoError(t, f.Close())
+				os.Stdout = oldStdout
+			}(os.Stdout)
+			os.Stdout = f
 
-	err = run([]string{fmt.Sprintf("-C=%s", tmpDir)})
-	require.Error(t, err, "expected no errors")
-	assert.EqualError(t, err, "found 5 issues")
+			// Pinning not to run into scoping errors.
+			gitDir := tt.gitDirArg
+			extraCmd := tt.extraCmd
+			err = run([]string{fmt.Sprintf(gitDir, tmpDir), extraCmd})
```

Comment:
So two things:

- the contents of gitDirArg don't vary
- the flag doesn't need to use the "-C=foo" form to be picked up

The suggestion from the first one would have been to just do:

```
run([]string{"-C="+tmpDir, extraCmd})
```

But also because you don't need to use `=`, tmpDir can just be the next arg, you can also do the following, which eliminates the string append too:

```suggestion
			err = run([]string{"-C", tmpDir, extraCmd})
```